docs/reference/settings/audit-settings.asciidoc

```diff
@@ -14,38 +14,26 @@ file. For more information, see
 
 `xpack.security.audit.enabled`::
 Set to `true` to enable auditing on the node. The default value is `false`.
-
-`xpack.security.audit.outputs`::
-Specifies where audit logs are output. For example: `[ index, logfile ]`. The
-default value is `logfile`, which puts the auditing events in a dedicated
-file named `<clustername>_audit.log` on each node.
-You can also specify `index`, which puts the auditing events in an {es} index
-that is prefixed with `.security_audit_log`. The index can reside on the same
-cluster or a separate cluster.
+This puts the auditing events in a dedicated file named `<clustername>_audit.log`
+on each node.
 +
-For backwards compatibility reasons, if you use the logfile output type, a
-`<clustername>_access.log` file is also created. It contains the same
-information, but it uses the older (pre-6.5.0) formatting style.
-If the backwards compatible format is not required, it should be disabled.
-To do that, change its logger level to `off` in the `log4j2.properties` file.
+For backwards compatibility reasons, a `<clustername>_access.log` file is also
```

Comment:
These are master docs, so I think this doesn't apply anymore?

Reply:
You're right 🙂  At first, this made me believe I still had a PR to do to remove the bwc audit format, that's how much I trust our docs!